Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -0,0 +1,300 @@
+                        guard let interfaceOrientation, initialCameraIsSet else { return }
+                        
+                        sceneViewProxy.updateCamera(
+                            frame: frame,
+                            cameraController: cameraController,
+                            orientation: interfaceOrientation,
+                            initialTransformation: .identity
+                        )
+                        sceneViewProxy.setFieldOfView(
+                            for: frame,
+                            orientation: interfaceOrientation
+                        )
+                    }
+                
+                if initialCameraIsSet {
+                    sceneViewBuilder(sceneViewProxy)
+                        .cameraController(cameraController)
+                        .attributionBarHidden(true)
+                        .spaceEffect(.transparent)
+                        .atmosphereEffect(.off)
+                        .interactiveNavigationDisabled(true)
+                        .onCameraChanged { camera in
+                            currentCamera = camera
+                        }
+                }
+                
+                if !coachingOverlayIsHidden {
+                    ARCoachingOverlay(goal: .geoTracking)
+                        .sessionProvider(arViewProxy)
+                        .active(coachingOverlayIsActive)
+                        .allowsHitTesting(false)
+                }
+            }
+            .observingInterfaceOrientation($interfaceOrientation)
+            .onAppear {
+                arViewProxy.session.run(configuration)
+            }
+            .onDisappear {
+                arViewProxy.session.pause()
+                Task { await locationDataSource.stop() }
+            }
+            .task {
+                do {
+                    try await locationDataSource.start()
+                    await withTaskGroup(of: Void.self) { group in
+                        group.addTask {
+                            for await location in locationDataSource.locations {
+                                lastLocationTimestamp = location.timestamp
+                                currentLocation = location
+                                await updateSceneView(for: location)
+                            }
+                        }
+                        group.addTask {
+                            for await heading in locationDataSource.headings {
+                                currentHeading = heading
+                                
+                                guard let currentCamera else { return }
+                                let camera = cameraController.originCamera.rotatedTo(
+                                    heading: calibrationHeading ?? heading,
+                                    pitch: currentCamera.pitch,
+                                    roll: currentCamera.roll
+                                )
```

Comment:
Sorry if I misled you here - as discussed with Mark, we don't need to set the heading when using ARKit - it constantly align the AR view with device heading with `gravityAndHeading` alignment.

Manual heading update would only be in these situations:

- Not using ARKit, which isn't supported in our new design
- Manually setting scene view heading in `local` mode. When the initial location is set by ARKit and then  stopped, we use the buttons to adjust heading in origin camera. We probably don't need the device heading update in this case either.

So I think the code can be omitted from this PR. Otherwise with current implementation the example's scene view will rotate twice the angle of the device rotation, causing the scene view to not align with the device camera.

```swift
group.addTask {
    for await heading in locationDataSource.headings {
        currentHeading = heading
        updateHeading(heading)
    }
}
```